coursemapper-kg/preprocess/Dockerfile

```
@@ -1,23 +1,35 @@
 FROM python:3.11
 
 # Set the working directory
 WORKDIR /app
 
-# Copy Pipfile and Pipfile.lock
-COPY Pipfile Pipfile.lock ./
+# Install APT packages
+ENV DEBIAN_FRONTEND=noninteractive
+ENV BUILD_DEPS="postgresql-common"
+ENV RUNTIME_DEPS="lbzip2 postgresql-client-16 rclone"
+RUN --mount=type=cache,sharing=private,target=/var/cache/apt \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)